common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -142,16 +142,19 @@ start_function() {
     [ "@BTL_TYPE@" = "LOCAL_BUS" ] && prplmesh_framework_init
     case "$PRPLMESH_MODE" in
         CA | ca)
-            prplmesh_platform_db_init "Multi-AP-Controller-and-Agent"
+            [ "@BWL_TYPE@" = "DUMMY" -a "$PLATFORM_INIT" = "true" ] && \
```

Comment:
The `"$PLATFORM_INIT" = "true"` part will cause the platform_db to not be initialized when we run our tests, because test_gw_repeater.sh skips platform initialisation. It's also not clear to me why you don't want to do this when PLATFORM_INIT is false - PLATFORM_INIT is about setting up the virtual networks for dummy operation...

Reply:
I dropped this commit